js/lui.js

```diff
@@ -17,7 +17,7 @@
 	// all the translations in one module
 	angular.module('lui.translates.userpicker', []);
 	angular.module('lui.translates.daterangepicker', []);
-	angular.module('lui.translates.tablegrid', []);
+	angular.module('lui.translates.tablegrid', ['lui.filters']);
```

Comment:
???

the module `'ui.translates.tablegrid` only contains the translations of the tablegrid, it has no reason to depend on `lui.filters`